docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,373 @@
+  - [Aiven Console](https://console.aiven.io/)
+  - [Aiven CLI client](/docs/tools/cli)
+  - [psql](https://www.postgresql.org/docs/current/app-psql.html) for advanced
+    configuration
+
+## Enable audit logging
+
+Enable audit logging by setting the `pgaudit.feature_enabled` parameter to
+`true` in your service's advanced configuration. Using the Aiven
+[console](https://console.aiven.io/), [CLI](/docs/tools/cli), or
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+
+<Tabs groupId="group1">
+<TabItem value="1" label="Aiven Console" default>
+
+:::important
+In [Aiven Console](https://console.aiven.io/), you can enable audit logging at the service
+level only. To enable it on a database or for a user's role, use
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+:::
+
+1. Log in to [Aiven Console](https://console.aiven.io/), and go to your organization
+   \> project > Aiven for PostgreSQL service.
+1. On the **Overview** page of your service, select <ConsoleLabel name="service settings"/>
+   from the sidebar.
+1. On the <ConsoleLabel name="service settings"/> page, go to the **Advanced configuration**
+   section and select **Configure**.
+1. In the **Advanced configuration** window, select **Add configuration options**, add
+   the `pgaudit.feature_enabled` parameter, set it to `true`, and select
+   **Save configuration**.
+
+</TabItem>
+<TabItem value="2" label="Aiven CLI">
+
+Use the [Aiven CLI client](/docs/tools/cli) to run the
+[avn service update](/docs/tools/cli/service-cli) command. Update your service by setting
+the `pgaudit.feature_enabled` parameter's value to `true`.
+
+```bash
+avn service update -c pgaudit.feature_enabled=true SERVICE_NAME
+```
+
+:::important
+By default, audit logging does not emit any audit records. To trigger a logging operation
+and start receiving audit records, configure audit logging parameters as detailed in
+[Configure audit logging](#configure-audit-logging).
+:::
+</TabItem>
+<TabItem value="3" label="psql">
+
+:::note
+psql allows for fine-grained enablement of audit logging: on a database, for a user's role,
+or for a database-role combination.
+:::
+
+#### Enable on a database
+
+:::important
+If you use PostgreSQL 14 or earlier, [upgrade](/docs/products/postgresql/howto/upgrade) to
+PostgreSQL 15 or later to enable `pgaudit` on your database.
+:::
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   SET pgaudit.feature_enabled='all';
+   ```
+
+#### Enable for a user's role
+
+:::important
+If you use PostgreSQL 14 or earlier, [upgrade](/docs/products/postgresql/howto/upgrade) to
+PostgreSQL 15 or later to enable `pgaudit` for a user's role.
+:::
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   SET pgaudit.feature_enabled='all';
```

Comment:
Same thing, it should be  `ALTER ROLE <role_name> SET pgaudit.feature_enabled = 'all'`